includes/html/pages/device/health.inc.php

```diff
@@ -38,7 +38,9 @@
     $component = new LibreNMS\Component();
     $components = $component->getComponents($device['device_id'], ['type'=> 'cisco-qfp']);
     $components = $components[$device['device_id']];
-    $qfp = count($components);
```

Comment:
Hi
You can replace line 41 with this : 
`$qfp = isset($components) ? count($components) : 0; `

Reply:
It's changed